CHANGES

```diff
@@ -1,8 +1,15 @@
 ===== dev =====
 
+====== Deprecations ======
+
+  * Lwt_main.yield and Lwt_unix.yield are deprecated in favor of the generic Lwt.pause, and Lwt_unix.auto_yield is deprecated in favor of the new Lwt_unix.auto_pause (#855, #858, Favonia).
+
+    Currently, Lwt.main resolves paused promises more frequently than yielded promises. The difference is unintended but existing applications could depend on it. Unifying the two pools of promises into one would eliminate possible discrepancies and simplify Lwt_main. As the first step, all yield-related functions are deprecated except for Lwt_main.abandon_yielded_and_paused for applications to work with existing libraries that still use yield. Once the migration has been completed, Lwt_main.abandon_yielded_and_paused will also be deprecated in favor of Lwt.abandon_paused.
```

Comment:
I'm don't know if the markup language used in this file (wikicreole) supports multiple-paragraph bullet-points. A conversion from that to markdown using pandoc seems to suggest it doesn't.

This could be shurnk and inlined. I'd say the two first sentences of the extra paragraph are the most important for the users of the library: it indicates the likely source of behaviour change. The rest is planning for Lwt maintainer (although it could be included in the documentation of `Lwt_main.abandon…`)

Reply:
I did not know it's in Creole---thought it's just a nicely formatted plain text file. I can move the text around.